python/ray/tests/test_actor_failures.py

```diff
@@ -910,83 +911,175 @@ def f():
     ):
         ray.get(f.remote())
 
-    """
-    Verify the basic case.
-    """
+
+def test_exit_actor_normal_actor_raise_immediately(shutdown_only, tmp_path):
```

Comment:
Can you add a test case for calling `ray.exit_actor()` within the constructor if there isn't one already?

Reply:
Added 3 tests: `test_exit_actor_normal_actor_in_constructor_should_exit`, `test_exit_actor_async_actor_in_constructor_should_exit`, and `test_exit_actor_async_actor_nested_task_in_constructor_should_exit`